openpracticelibrary/content-api/base/configMap.yaml

```diff
@@ -5,3 +5,5 @@ metadata:
 data:
   nodeEnv: "production"
   dbHost: "opl-mongo"
+  oplContentApiBaseUrl: "http://opl-content-api-opl-dev.apps.s44.core.rht-labs.com"
```

Comment:
As this changes per environment, we should patch the configmap in the `overlays` directory

Reply:
There's already a patchCM.yaml file that patches the configmap for each environment in overlays.  Here's one:

https://github.com/openpracticelibrary/opl-cd/blob/master/openpracticelibrary/content-api/overlays/development/patchCM.yaml

Reply:
Thanks @shaheinm, Added both email and baseUrl to patchCM files since I expect them both to change with the environment stage.  For example, we soon won't want test emails going to our production email account.  See latest commit.